src/react-app-env.d.ts

```diff
@@ -1 +1,47 @@
 /// <reference types="react-scripts" />
+
+interface Comment {
+  postId: number;
+  id: number;
+  name: string;
+  email: string;
+  body: string;
+}
+
+interface Geo {
+  lat: string;
+  lng: string;
+}
+
+interface Company {
+  name: string;
+  catchPhrase: string;
+  bs: string;
+}
+
+interface Address {
+  street: string;
+  suite: string;
+  city: string;
+  zipcode: string;
+  geo: Geo;
+}
```

Comment:
I think this might be redundant, since you don't display all these props

Reply:
as Dima said, it's better to describe everything at the beginning